var/spack/repos/builtin/packages/podio/package.py

```diff
@@ -38,6 +41,7 @@ class Podio(CMakePackage):
     depends_on('python', type=('build', 'run'))
     depends_on('py-pyyaml', type=('build', 'run'))
     depends_on('py-jinja2@2.10.1:', type=('build', 'run'), when='@0.12.0:')
+    depends_on('sio', type=('build', 'run'), when='@0.13.0: +sio')
 
     def cmake_args(self):
         args = ['-DBUILD_TESTING=%s' % self.run_tests, ]
```

Comment:
```suggestion
        args = [
            self.define('BUILD_TESTING', self.run_tests),
            self.define_from_variants('ENABLE_SIO', 'sio'),
        
          ]
        
```

Reply:
Thanks for catching this. Completely forgot about propagating this.